src/main/java/ch/njol/skript/expressions/ExprTime.java

```diff
@@ -74,7 +74,7 @@ public Class<?>[] acceptChange(final ChangeMode mode) {
 
 	@Override
 	public void change(Event event, Object @Nullable [] delta, ChangeMode mode) {
-		if (getExpr() == null || delta == null)
```

Comment:
this still needs to exist
getExpr CAN be null, if something went wrong

though i suppose it could be argued if something went that wrong, a stacktrace is probably a good thing

Reply:
this is the only occurrence of getExpr getting null-checked in the entirety of skript lol

Reply:
fair enough lol